packages/common-cosmos/package.json

```diff
@@ -13,17 +13,30 @@
   "main": "dist/index.js",
   "license": "GPL-3.0",
   "dependencies": {
+    "@cosmology/telescope": "^0.104.0",
+    "@protobufs/cosmos": "^0.1.0",
+    "@protobufs/cosmos_proto": "^0.0.10",
+    "@protobufs/cosmwasm": "^0.1.1",
+    "@protobufs/gogoproto": "^0.0.10",
+    "@protobufs/google": "^0.0.10",
+    "@protobufs/ibc": "^0.1.0",
+    "@protobufs/tendermint": "^0.0.10",
     "@subql/common": "^2.4.0",
     "@subql/types-cosmos": "workspace:*",
+    "fs-extra": "^11.1.1",
     "js-yaml": "^4.1.0",
+    "protobufjs": "^7.2.4",
```

Comment:
Are you able to check compatibility with this please. Cosmjs supports a different version and dependabot created this PR https://github.com/subquery/subql-cosmos/pull/143

Reply:
We only seem to use it for the `Root` type as well. Maybe we don't need the explicit dependency and it would be covered by transient dependencies